### PR TITLE
Lytesttools example doc

### PR DESCRIPTION
**Pull request by evanchia-ly-sdets**
Opening a new PR due to the fork change.  LyTestTools example test documentation.

### CODE REVIEW

**Review by Kadino — CHANGES_REQUESTED** [review]

File: content/docs/user-guide/testing/lytesttools/example-test-walkthrough.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+* Completed [Getting Started](getting-started.md)
+* A properly configured [Open 3D Engine build](/docs/user-guide/build/)
+* Open the sample test file (`Tools/LyTestTools/tests/integ/sanity_tests.py`)
+* Basic understanding of the [Pytest](https://pytest.org) module
+
+## Importing Dependencies
+
+Import any desired LTT modules from the package `ly_test_tools`. All LTT modules can be viewed at `Tools/LyTestTools/ly_test_tools`. The `launchers.launcher_helper` module helps create Launcher objects which control the Open 3D Engine (O3DE) Editor and game clients. The `builtin.helpers` module helps create the Workspace object, which controls the testing workspace in LTT. The `environment` module contains tools that involve the system's environment such as processes or timed waiters.
+
+```python
```
[the remark this answers is not in the document]
Comment (evanchia-ly-sdets) [reply content/docs/user-guide/testing/lytesttools/example-test-walkthrough.md]: moved explanations to inline comments

File: content/docs/user-guide/testing/lytesttools/example-test-walkthrough.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+---
+description: ' Walk through and explanation of the LyTestTools example test. '
+title: LyTestTools Example Test Walkthrough
+linktitle: LyTestTools Example Test
+---
+
+Instructions on how to get started with an example test using the LyTestTools (LTT) testing package.
```
Comment (AMZN-scspaldi) [inline content/docs/user-guide/testing/lytesttools/example-test-walkthrough.md:7]: This isn't a complete sentence. Maybe, "These are instructions..."?

File: content/docs/user-guide/testing/lytesttools/example-test-walkthrough.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+## Prerequisites
+
+* Completed [Getting Started](getting-started.md)
+* A properly configured [Open 3D Engine build](/docs/user-guide/build/)
+* Open the sample test file (`Tools/LyTestTools/tests/integ/sanity_tests.py`)
+* Basic understanding of the [Pytest](https://pytest.org) module
+
+Make sure to complete all of the Prerequisite steps above. The provided example test contains in line comments to walk the user through each step. Open the the sample test and follow along for a complete walkthrough.
```
Comment (AMZN-scspaldi) [inline content/docs/user-guide/testing/lytesttools/example-test-walkthrough.md:16]: nit: should be lower case

Comment (AMZN-scspaldi) [inline content/docs/user-guide/testing/lytesttools/example-test-walkthrough.md:16]: Should this be 'in-line'?

**Review by AMZN-scspaldi — CHANGES_REQUESTED** [review]

File: content/docs/user-guide/testing/lytesttools/example-test-walkthrough.md (hunk 2 above)
Comment (evanchia-ly-sdets) [reply content/docs/user-guide/testing/lytesttools/example-test-walkthrough.md:7]: fixed

File: content/docs/user-guide/testing/lytesttools/example-test-walkthrough.md (hunk 3 above)
Comment (evanchia-ly-sdets) [reply content/docs/user-guide/testing/lytesttools/example-test-walkthrough.md:16]: fixed

Comment (evanchia-ly-sdets) [reply content/docs/user-guide/testing/lytesttools/example-test-walkthrough.md:16]: I believe "inline" is the correct term for this case

**Review by AMZN-scspaldi — APPROVED** [review]

File: content/docs/user-guide/testing/lytesttools/example-test-walkthrough.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+These are instructions on how to get started with an example test using the LyTestTools (LTT) testing package.
+
+## Prerequisites
+
+* Completed [Getting Started](getting-started.md)
+* A properly configured [Open 3D Engine build](/docs/user-guide/build/)
+* Open the sample test file (`Tools/LyTestTools/tests/integ/sanity_tests.py`)
+* Basic understanding of the [Pytest](https://pytest.org) module
+
+Make sure to complete all of the prerequisite steps above. The provided example test contains in line comments to walk the user through each step. Open the the sample test and follow along for a complete walkthrough.
```
Comment (sptramer) [inline content/docs/user-guide/testing/lytesttools/example-test-walkthrough.md:16]: ? This looks incomplete.

**Review by sptramer — CHANGES_REQUESTED** [review]
@evanchia-ly-sdets - Do you still want this PR merged or should it be closed and worked on more? It looks like the new content (test walkthrough) is incomplete.

File: content/docs/user-guide/testing/lytesttools/example-test-walkthrough.md (hunk 4 above)
Comment (evanchia-ly-sdets) [reply content/docs/user-guide/testing/lytesttools/example-test-walkthrough.md:16]: The walkthrough has been moved to be in line on the example test itself. This document covers set up and directs the user to  continue the walkthrough in the test.
https://github.com/o3de/o3de/pull/1979/files

Comment (Kadino) [reply content/docs/user-guide/testing/lytesttools/example-test-walkthrough.md:16]: The page is now a bit curt and awkward to read through. Opening the sample test file should probably not be in 'prerequisites' as it is part of walking through the example. There could also be a brief outline in plain English to suggest which lessons the user should learn from reading the file.

Comment (evanchia-ly-sdets) [reply content/docs/user-guide/testing/lytesttools/example-test-walkthrough.md]: Added more description

File: content/docs/user-guide/testing/lytesttools/example-test-walkthrough.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+
+These are instructions on how to get started with an example test using the LyTestTools (LTT) testing package.
+
+## Prerequisites
+
+* Completed [Getting Started](getting-started.md)
+* A properly configured [Open 3D Engine build](/docs/user-guide/build/)
+* Basic understanding of the [Pytest](https://pytest.org) module
+
+Make sure to complete all of the prerequisite steps above. Open the sample test file located at `Tools/LyTestTools/tests/integ/sanity_tests.py`. The provided example test contains in line comments to walk the user through each step. Following this walkthrough will instruct the user how to import the LyTestTools package, interact with Editor and game launcher classes, and structure tests.
```
Comment (Kadino) [inline content/docs/user-guide/testing/lytesttools/example-test-walkthrough.md:15]: Should probably add o3de to ```"o3de/Tools/LyTestTools/tests/integ/sanity_tests.py"```

**Review by Kadino — APPROVED** [review]

File: content/docs/user-guide/testing/lytesttools/example-test-walkthrough.md (hunk 5 above)
Comment (evanchia-ly-sdets) [reply content/docs/user-guide/testing/lytesttools/example-test-walkthrough.md:15]: I believe the convention is to assume the path starts with o3de?

Comment (sptramer) [reply content/docs/user-guide/testing/lytesttools/example-test-walkthrough.md:15]: This is correct, paths are assumed to be relative to the O3DE clone.

**Review by sptramer — CHANGES_REQUESTED** [review]
There's only a "prerequisites" section in the walkthrough here - this PR should be moved to WIP or closed if it won't be completed.

**Comment by sptramer** [comment]
Closing after a discussion with SDETs - we're going to try and integrate this with the Python documentation itself and then publish it in conjunction with testing platform API ref.
